pom.xml

```diff
@@ -178,6 +180,7 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <version>3.0.0-M4</version>
           <configuration>
+            <failIfNoTests>false</failIfNoTests>
```

Comment:
Why?

Reply:
This allows to quickly only run 1 or few specific tests in CI by adding -Dtest=... to the cli in the workflow file without requiring additional parameters or tweaks in the yaml file

Reply:
The situation you describe is adding an option to the CI workflow -- it's simple to add the `failIfNoTests` option as well.  I'd prefer _not_ to have "just in case" options in the POM.

Reply:
Ok, if you prefer I'll remove.
Different opinion. I prefer making dev life quicker and easier when I can. I just see more value in having it than not having it. Like the commented things and such because it's not everybody that is maven fluent and GA fluent... Helping peers being more efficient has some value.

Reply:
I'd prefer it be removed -- it doesn't really belong as a project-level property value.  If someone needs it, either in a CI run or locally, they can set the value.

Reply:
done